discovery-infra/test_infra/helper_classes/config/controller_config.py

```diff
@@ -0,0 +1,33 @@
+from abc import ABC
+from pathlib import Path
+from dataclasses import dataclass
+
+from test_infra import consts
+from .base_config import _BaseConfig
+
+
+@dataclass
+class ControllerConfig(_BaseConfig, ABC):
+    platform: str = None
+    base_dns_domain: str = None   # Todo - Might change during MGMT-5370
+    is_ipv6: bool = None   # Todo - Might change during MGMT-5370
+    bootstrap_in_place: bool = None
+    private_ssh_key_path: Path = None
+    working_dir: str = consts.WORKING_DIR
+
+    master_memory: int = None
+    master_vcpu: int = None
+    masters_count: int = None
+    master_cpu_mode: str = None
+    master_disk: int = None   # disk size in MB.
+    master_disk_size_gib: str = None # disk size in GB.
```

Comment:
it doesn't sound fun to have two sources of truth
maybe save the int one, and have a small function that returns the other value?